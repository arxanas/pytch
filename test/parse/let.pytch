let foo = 1
let foo bar =
  let baz = 3 in
  bar
