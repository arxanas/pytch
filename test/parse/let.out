(Program
  (LetPlainStmt
    (Ident 'foo')
    (IntLiteral 1))
  (LetFuncStmt
    (Ident 'foo')
    (Ident 'bar')
    (LetInExpr
      (LetPlainStmt
        (Ident 'baz')
        (IntLiteral 3))
      (Ident 'bar'))))
